infrastructure/main.tf

```diff
@@ -9,4 +9,31 @@ data "azurerm_key_vault" "sscs_key_vault" {
 
 locals {
   azureVaultName = "sscs-${var.env}"
-}
\ No newline at end of file
+  sendgrid_subscription = {
+    prod    = "8999dec3-0104-4a27-94ee-6588559729d1"
+    nonprod = "1c4f0704-a29e-403d-b719-b90c34ef14c9"
+  }
+}
+
+data "azurerm_key_vault" "send_grid" {
```

Comment:
I think we should move the infra as a separate PR as we will already have the secrets in vault and build might fail. may be we will need to define secrets with new name but just a thought to do it as a separate PR ?

Reply:
Sure. I will move these out to separate PR for infra